virttest/qemu_storage.py

```diff
@@ -348,7 +349,7 @@ def snapshot_del(self, blkdebug_cfg=""):
         else:
             cmd += " %s" % self.image_filename
 
-        process.system_output(cmd)
+        return process.run(cmd, ignore_status=True)
 
```

Comment:
You set ignore_status inside of snapshot_del.
It's better to check no bad influence on all of it's caller.